ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -24,12 +24,12 @@ releases:
     candidate:
       product: ocp
       relative: 1
-      stream: ci
+      stream: nightly
```

Comment:
Why is this changing? It's not obvious to me from the commit message or PR summary.

Reply:
We typically set the release stream to "nightly" to reduce noise on maintenance branches, as "ci" is the unfiltered firehose resulting from building each commit of the branch